src/header.ts

```diff
@@ -40,12 +50,12 @@ export interface CoreHeader extends FormatMixins {
   /**
    * The anchor position for placing the title. One of `"start"`, `"middle"`, or `"end"`. For example, with an orientation of top these anchor positions map to a left-, center-, or right-aligned title.
    */
-  titleAnchor?: TitleAnchor;
+  titleAnchor?: TitleAnchor; // We don't allow signal for titleAnchor since there is a dependent logic
```

Comment:
What does this mean? Can you point to the logic and what it depends on?

Reply:
It means we read the value of titleAnchor to determine other values. 

> Can you point to the logic and what it depends on?

Too much work. One can always add `| SignalRef` here and see where it breaks.

Reply:
ok